### PR TITLE
mixMunsell patch

### PR DESCRIPTION
**Pull request by brownag**
I noticed a couple small things -- edge cases -- while messing around with the Harden color index functions on the `jacobs2000` data -- added some tests to the master branch and I fix them here -- this is the one conflicting file.

### CODE REVIEW

File: R/mixMunsell.R [hunk 1]
```diff
@@ -78,66 +78,97 @@
...
-    
-  # x and w same lengths
-  
-  # x contains valid Munsell
-  if(any(is.na(parseMunsell(x)))) {
-    stop('input must be valid Munsell notation, neutral hues not supported')
+
+  # can't mix a single color, just give it back at 0 distance
+  if (length(unique(x)) == 1) {
+    return(data.frame(munsell = x, distance = 0))
```
Comment (brownag) [inline R/mixMunsell.R:122]: `unique(x)`

File: R/mixMunsell.R [hunk 2]
```diff
@@ -149,52 +180,54 @@ mixMunsell <- function(x, w = rep(1, times = length(x)) / length(x), n = 1) {
...
     munsell = names(m.match),
     distance = m.match,
     stringsAsFactors = FALSE
   )
-  
+
   # clean-up row names
-  row.names(res) <- as.character(1:nrow(res))
-  
+  row.names(res) <- NULL
```
Comment (dylanbeaudette) [inline R/mixMunsell.R:228]: Hey that is cool, learned something new.

**Review by dylanbeaudette — APPROVED** [review]
Thanks for improving this function and tests.

File: R/mixMunsell.R (hunk 2 above)
Comment (brownag) [reply R/mixMunsell.R:228]: It is cool, save a couple key strokes. 

Not that it matters for this function but non-`NULL` `row.names<-` to a [`tibble` causes warnings]( https://tibble.tidyverse.org/reference/rownames.html). I think the thought is that they get do get messy, as above, and if they contain any real info they should be a column.

Comment (dylanbeaudette) [reply R/mixMunsell.R:228]: Good point. I'll be sure to use this construct moving forward.
